tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot

```diff
@@ -0,0 +1,180 @@
+
+    user opens details dropdown    %{TEST_TOPIC_NAME}
+    user waits until details dropdown contains publication    %{TEST_TOPIC_NAME}    ${PUB_NAME_ARCHIVE}
+    user checks publication bullet contains link    ${PUB_NAME_ARCHIVE}    View statistics and data
+
+    user clicks element    testid:View stats link for ${PUB_NAME_ARCHIVE}
+
+    user waits until h1 is visible    ${PUB_NAME_ARCHIVE}    %{WAIT_MEDIUM}
+    user waits until page contains    This is the latest data
+    ${PUB_ARCHIVE_URL}=    user gets url
+    set global variable    ${PUB_ARCHIVE_URL}
+
+Navigate to superseding-publication release on admin site
+    user navigates to editable release summary from admin dashboard    ${PUB_NAME_SUPERSEDE}
+    ...    ${RELEASE_NAME_SUPERSEDE} (not Live)
+
+Import superseding-publication subject to release
+    user uploads subject    ${SUBJECT_NAME_SUPERSEDE}    upload-file-test.csv    upload-file-test.meta.csv
+
+Add data guidance to superseding-publication subject
+    user clicks link    Data guidance
+    user waits until h2 is visible    Public data guidance
+
+    user waits until page contains element    id:dataGuidance-dataFiles
+    user waits until page contains accordion section    ${SUBJECT_NAME_SUPERSEDE}
+    user enters text into data guidance data file content editor    ${SUBJECT_NAME_SUPERSEDE}
+    ...    ${SUBJECT_NAME_SUPERSEDE} data guidance content
+
+    user clicks button    Save guidance
+    user waits until page contains button    Edit guidance
+
+Go to "Sign off" page and approve superseding-publication release
+    user clicks link    Sign off
+    user approves original release for immediate publication
+
+Check public Find stats page, that archive-publication is now archived and superseding-publication appears
+    user navigates to find statistics page on public frontend
+
+    user waits until page contains accordion section    %{TEST_THEME_NAME}
+    user opens accordion section    %{TEST_THEME_NAME}
+    user waits until accordion section contains text    %{TEST_THEME_NAME}    %{TEST_TOPIC_NAME}
+
+    user opens details dropdown    %{TEST_TOPIC_NAME}
+    user checks page does not contain    ${PUB_NAME_ARCHIVE}
+
+    user waits until details dropdown contains publication    %{TEST_TOPIC_NAME}    ${PUB_NAME_SUPERSEDE}
+    user checks publication bullet contains link    ${PUB_NAME_SUPERSEDE}    View statistics and data
+
+Check public superseding-publication release page displays correctly
+    user clicks element    testid:View stats link for ${PUB_NAME_SUPERSEDE}
+
+    user waits until h1 is visible    ${PUB_NAME_SUPERSEDE}    %{WAIT_MEDIUM}
+    user waits until page contains    This is the latest data
+
+Check public archive-publication release page displays correctly
+    go to    ${PUB_ARCHIVE_URL}
+    user waits until h1 is visible    ${PUB_NAME_ARCHIVE}    %{WAIT_MEDIUM}
+    user checks page does not contain    This is the latest data
```

Comment:
Not related to this test but on Permalink pages we show a warning a soon as the Permalink is *not* the latest data, i.e. not from the latest published release.

See the code in `Data.Api.Services.PermalinkService.BuildViewModel` which sets that flag.

If a Permalink table is for the latest published release of that publication, but that publication is now superseded, should we now show the same warning?

Reply:
Discussed this with Ben on Slack. Made the change to ensure permalinks under superseded publications now show an out-of-date warning. Also updated unit tests and add UI tests to check permalinks work correctly around this change.